py3status/modules/wifi.py

```diff
@@ -63,16 +64,44 @@ class Py3status:
     cache_timeout = 10
     device = 'wlan0'
     down_color = 'bad'
-    format_down = 'W: down'
-    format_up = 'W: {bitrate} {signal_percent} {ssid}'
+    format = 'W: [{bitrate} {signal_percent} {ssid}|\?show down]'
```

Comment:
This makes me worry that you are not testing things.

Surely this is something that you can test. but the if may well be needed.